app/client/templates/elements/selectableUnit.js

```diff
@@ -37,6 +37,25 @@ selectableUnits = [{
     value: 'eur'
 }];
 
+/* Aprils fool */
+if (moment().format('MM-DD')=='03-01') {
+    selectableUnits.push({
+                text: 'NOETHER', value: 'noether'
```

Comment:
Emmy Noether is a very important mathematician and I'm showing this as a easter egg on april 01, a week before her birthday